pkg/util/apptainerconf/config.go

```diff
@@ -530,4 +532,10 @@ download buffer size = {{ .DownloadBufferSize }}
 # Whether to use systemd to manage container cgroups. Required for rootless cgroups
 # functionality. 'no' will manage cgroups directly via cgroupfs.
 systemd cgroups = {{ if eq .SystemdCgroups true }}yes{{ else }}no{{ end }}
+
+# APPTHEUS SOCKET PATH: [STRING]
+# DEFAULT: /run/apptheus/gateway.sock
```

Comment:
Doesn't it require root to create that?

Reply:
> Doesn't it require root to create that?

this sock is created by apptheus, the new tool, apptheus is started with privileges as it will manipulate cgroup. Apptainer just needs to try connecting.